creator-node/src/fileManager.js

```diff
@@ -463,6 +461,54 @@ function getFileExtension (fileName) {
   return (fileName.lastIndexOf('.') >= 0) ? fileName.substr(fileName.lastIndexOf('.')).toLowerCase() : ''
 }
 
+/**
+ * Checks the file type. Throws an error if not accepted.
+ * @param {Object} req express request object
```

Comment:
can we pass in logger here instead of req?

Reply:
+1

Reply:
yes, will update